### DISCUSSION TITLE
Punisher Capture

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#9294] [Q&A] [answered]
I did the Quarg/Coalition exploit, captured a Punisher, and saw that most of Coalition space was restricted to me. Is this temporary or permanent, and can I do stuff in-game to fix this?
![image](https://github.com/endless-sky/endless-sky/assets/104505171/15110e08-19f8-4ca1-bf08-b533783e8af5)

**Comment by lumbar527** [comment, marked as answer]
I think you're stuck ~hostile~ restricted without cheating.

You might just be stuck in Coalition space altogether—unless, can you land somewhere with an outfitter?

**Reply by SultanRTG** [reply]
It's not hostile, it just says "restricted".

**Reply by Saugia** [reply]
Have you made contact with the Coalition yet? If not, and if you have not done missions for them yet, they start off restricted in many places.

**Reply by SultanRTG** [reply]
I made contact, did all 30 missions, and got to a point where i could take a punisher without retribution.

**Reply by SultanRTG** [reply]
It sorta went away.
